components/n-ui/tour-tip/template.html

```diff
@@ -10,19 +10,11 @@
 			{{#if data.content.body}}
 				<p class="tour-tip__body">{{{data.content.body}}}</p>
 			{{/if}}
-			{{#ifAll data.content.ctas data.content.ctas.length}}
-				{{#if data.settings.isAnon}}
-					{{#unless data.settings.hideCtaFromAnon}}
-						{{#each data.content.ctas}}
-							<a href="{{{url}}}" class="tour-tip__cta o-buttons o-buttons--standout" data-trackable="cta">{{label}}</a>
-						{{/each}}
-					{{/unless}}
-				{{else}}
-					{{#each data.content.ctas}}
-						<a href="{{{url}}}" class="tour-tip__cta o-buttons o-buttons--standout" data-trackable="cta">{{label}}</a>
-					{{/each}}
-				{{/if}}
-			{{/ifAll}}
+			{{#if data.content.ctas}}
```

Comment:
wasn't sure whether to go for if anon remove cta property altogether or to make it an empty array. Thought the easiest would be just to remove the property